pkg/rancher-desktop/window/preferences.ts

```diff
@@ -62,6 +62,38 @@ export function openPreferences() {
       },
       'preferences help',
     );
+
+    preferencesNavItems.forEach(({ name }, index) => {
+      Shortcuts.register(
+        window,
+        {
+          ...CommandOrControl,
+          key: index + 1,
+        },
+        () => window.webContents.send('route', { name }),
+        `switch preferences tabs ${ name }`,
+      );
+    });
+
+    Shortcuts.register(
+      window,
+      {
+        ...CommandOrControl,
+        key: ']',
+      },
+      () => window.webContents.send('route', { direction: 'forward' }),
+      'switch preferences tabs by cycle [forward]',
+    );
+
+    Shortcuts.register(
+      window,
+      {
+        ...CommandOrControl,
+        key: '[',
+      },
+      () => window.webContents.send('route', { direction: 'back' }),
+      'switch preferences tabs by cycle [back]',
+    );
```

Comment:
I realize it's not part of this PR, but it's the first time I've looked at this code, and the `if ... else` block at the end of `isRegistered` should be replaced with `return !!this.windows[id];`

Reply:
Fixed